src/config/dynamic-theme-fixes.config

```diff
@@ -933,6 +933,15 @@ CSS
 
 ================================
 
+inoreader.com
+
+CSS
+div.article_expanded{
```

Comment:
`div` selector part is unnecessary, `.article_expanded` should be enough. Please put ` ` (space) before `{`.